File: Android/PrathamBulani_PrathamBulani_2024cs1107/GettingStarted/print_all_primes_till_n.java
import java.util.*;

public class Main {
  public static void main(String[] args) {
    Scanner scn = new Scanner(System.in);
    int low = scn.nextInt();
    int high = scn.nextInt();
    for (int num = low; num <= high; num++) {
      int div = 2;
      while (div * div <= num) {
        if (num % div == 0) {
          break;
        }
        div++;
      }
      if (div * div > num) {
        System.out.println(num);
      }
    }
  }
}

File: Android/iamAditya-07_AdityaPratapSingh_2024cse1115_2/GettingStarted/PrintAllPrimesTillN.cpp
#include<iostream>
using namespace std;
int main() {
    
    int low;
    cin>>low;
    
    int high;
    cin>>high;
    
    for(int n=low;n<=high;n++){
        int div=2;
        while(div*div<=n){
            if(n%div==0){
                break;
            }
            div++;
        }
        if(div*div>n){
            cout<<n;
            cout<<"\n";
        }
    }

}